src/app.ts

```diff
@@ -12,6 +12,8 @@ import {
 import { autoUpdater } from 'electron-updater'
 import utils, { is } from 'electron-util'
 import log from 'electron-log'
+import electronDl from 'electron-dl'
+import electronContextMenu from 'electron-context-menu'
```

Comment:
Does this conflict with the context menu in gmail when right clicking on an email?

![2019-04-29_17-56-47](https://user-images.githubusercontent.com/25914066/56932145-2b98e080-6aa8-11e9-91d3-b7b8731a70da.png)

Reply:
Can confirm that it doesn't conflict.